graph/load.go

```diff
@@ -68,7 +68,7 @@ func (s *TagStore) CmdLoad(job *engine.Job) engine.Status {
 
 		for imageName, tagMap := range repositories {
 			for tag, address := range tagMap {
-				if err := s.Set(imageName, tag, address, true); err != nil {
+				if err := s.SetLoad(imageName, tag, address, true, job.Stdout); err != nil {
```

Comment:
Sorry but I don't get it: if `SetLoad` returns an error, why isn't it reported to the client? Why do we need to pass the `job.Stdout` and emit a new line of output there, isn't the solution just to write the error result to `w` in `postImagesLoad`?